docs/_documentations/debugging.md

```diff
@@ -22,8 +22,10 @@ Complete the following steps to debug in Codewind applications:
 Remote debugging is the same as local debugging with the following exceptions:
 - Install `kubectl` or `oc` and make sure you are logged in to your cluster.
 - After the project restarts in debug mode, Codewind automatically sets up a port forward from the application's debug port to a port on your local machine.
-   - In Eclipse, a launch is created for the port forward that appears in the Debug view.
+**Starting a debug session**
+   - In Eclipse Node.js projects, a launch is created for the port forward that appears in the Debug view. In Eclipse Java projects, the port forward is added as a process to the debug launch instead of a separate port forward launch.
```

Comment:
How about:
In Eclipse, for Node.js projects, a launch is created for the port forward that appears in the Debug view. For Java projects, the port forward is added as a process to the debug launch.